src/api_routes_v4.rs

```diff
@@ -0,0 +1,389 @@
+          .route("/delete", web::post().to(delete_post))
+          .route("/remove", web::post().to(remove_post))
+          .route("/mark_as_read", web::post().to(mark_post_as_read))
+          .route("/mark_many_as_read", web::post().to(mark_posts_as_read))
+          .route("/hide", web::post().to(hide_post))
+          .route("/lock", web::post().to(lock_post))
+          .route("/feature", web::post().to(feature_post))
+          .route("/list", web::get().to(list_posts))
+          .route("/like", web::post().to(like_post))
+          .route("/like/list", web::get().to(list_post_likes))
+          .route("/save", web::put().to(save_post))
+          .route("/report", web::post().to(create_post_report))
+          .route("/report/resolve", web::put().to(resolve_post_report))
+          .route("/report/list", web::get().to(list_post_reports))
+          .route("/site_metadata", web::get().to(get_link_metadata)),
+      )
+      // Comment
+      .service(
+        // Handle POST to /comment separately to add the comment() rate limitter
+        web::resource("/comment")
+          .guard(guard::Post())
+          .wrap(rate_limit.comment())
+          .route(web::post().to(create_comment)),
+      )
+      .service(
+        web::scope("/comment")
+          .route("", web::get().to(get_comment))
+          .route("", web::put().to(update_comment))
+          .route("/delete", web::post().to(delete_comment))
+          .route("/remove", web::post().to(remove_comment))
+          .route("/mark_as_read", web::post().to(mark_reply_as_read))
+          .route("/distinguish", web::post().to(distinguish_comment))
+          .route("/like", web::post().to(like_comment))
+          .route("/like/list", web::get().to(list_comment_likes))
+          .route("/save", web::put().to(save_comment))
+          .route("/list", web::get().to(list_comments))
+          .route("/report", web::post().to(create_comment_report))
+          .route("/report/resolve", web::put().to(resolve_comment_report))
+          .route("/report/list", web::get().to(list_comment_reports)),
+      )
+      // Private Message
+      .service(
+        web::scope("/private_message")
+          .route("/list", web::get().to(get_private_message))
+          .route("", web::post().to(create_private_message))
+          .route("", web::put().to(update_private_message))
+          .route("/delete", web::post().to(delete_private_message))
+          .route("/mark_as_read", web::post().to(mark_pm_as_read))
+          .route("/report", web::post().to(create_pm_report))
+          .route("/report/resolve", web::put().to(resolve_pm_report))
+          .route("/report/list", web::get().to(list_pm_reports)),
+      )
+      // User
+      .service(
+        web::scope("/account/auth")
+          .guard(guard::Post())
+          .wrap(rate_limit.register())
+          .route("register", web::post().to(register))
+          .route("login", web::post().to(login))
```

Comment:
The account auth register is probably failing because these and a bunch of other routes need `/` before.

Reply:
Thanks that was it. If only it would show a warning or error for this.